testing/unittests/fxpptests.cpp

```diff
@@ -44,86 +44,28 @@ static IPTree* getPTree(const char* test, const char* subSection, const char* xm
   }
   catch (IException* e)
   {
-    StringBuffer msg;
-    msg.appendf("Test(%s", test);
+    StringBuffer msg("Exception creating ");
     if (subSection)
-      msg.appendf("/%s", subSection);
-    msg.append(") exception creating PTree: ");
+      msg.appendf("'%s' PTree: ", subSection);
+    else
+      msg.append("PTree: ");
```

Comment:
Change to unconditionally add `PTree: `.

Reply:
Do you mean to eliminate output of the subSection name? I thought that would be helpful in identifying the 'sub-test' for the function.

Reply:
simplified